OpenUpSA/wazimap-ng-ui
# removing hardcoded gcro config

## Description
Removed the hardcoded gcro config so the configurations that are sent through the API will be used instead

## Related Issue


## How to test it locally
* Navigate to `/#geo:EKU` on `gcro.openup.org.za` instance(this could be accomplished by hardcoding the `hostname` parameter in `index.js`)
* Hardcoded configuration only let `municipality` children to be drawn. So check if the `planning_region` children geographies are drawn too. 

## Screenshots
Before : 
![image](https://user-images.githubusercontent.com/53019884/95191217-1960bd00-07d9-11eb-89a9-d417f46db491.png)

After : 
![image](https://user-images.githubusercontent.com/53019884/95191189-0cdc6480-07d9-11eb-900c-a135bbe0c110.png)


## Changelog

### Added

### Updated
* Modified `index.js` to use `config` parameter instead of the hardcoded configuration
* Added null check to `profile_header.js` to prevent console error when there are no facilities for the current geography

### Removed
* configurations > geography_gcro.js and it's references to remove hardcoded configuration


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
